alb/terraform-examples/nsxt-cloud-deployment/variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "avi_username" {
+  type    = string
+  default = <Avi Username>
```

Comment:
These should be `description` vs `default` in my opinion. Simply remove the `default`.